### PR TITLE
Commerce-5392 Manage cart endopoint permissions for guest user

### PR DESCRIPTION
**Pull request by luca-pellizzon**

### CODE REVIEW

**Comment by luca-pellizzon** [comment]
ci:test:sf

**Comment by luca-pellizzon** [comment]
ci:test:relevant

**Comment by luca-pellizzon** [comment]
ci:test:sf

**Comment by luca-pellizzon** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-api/src/main/java/com/liferay/commerce/constants/CommerceSAPConstants.java [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+			"CartItemResourceImpl";
+
+	public static final String CLASS_NAME_COMMERCE_HEADLESS_CART_RESOURCE =
+		"com.liferay.headless.commerce.delivery.cart.internal.resource.v1_0." +
+			"CartResourceImpl";
+
+	public static final String CLASS_NAME_COMMERCE_SEARCH_RESOURCE =
+		"com.liferay.commerce.frontend.internal.search.CommerceSearchResource";
+
+	public static final String COMMERCE_SAP_ENTRY_NAME = "COMMERCE_DEFAULT";
```
Comment (dsitu) [inline modules/apps/commerce/commerce-api/src/main/java/com/liferay/commerce/constants/CommerceSAPConstants.java:43]: @luca-pellizzon I think we should just call this 
```suggestion
	public static final String _SAP_ENTRY_NAME = "COMMERCE_DEFAULT";
```

**Review by dsitu — CHANGES_REQUESTED** [review]

**Comment by jpince** [comment]
@dsitu would you be able to make the change and push this forward? We'd like to keep this moving since [COMMERCE-5392](https://issues.liferay.com/browse/COMMERCE-5392) / [COMMERCE-5672](https://issues.liferay.com/browse/COMMERCE-5672) is a blocker for 7.3 FP1.

**Comment by dsitu** [comment]
Added SF and opened this PR https://github.com/liferay-commerce/liferay-portal/pull/623

**Comment by riccardo-alberti** [comment]
closing this PR. Please refer to #623

**Pull request closed without merging**